packages/vcvrack-ahornberg/PKGBUILD

```diff
@@ -0,0 +1,42 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Florian Hülsmann <fh@cbix.de>
+
+_slug=Ahornberg
+_name=Ahornberg-VCV-Modules
+pkgname=vcvrack-ahornberg
+pkgver=2.1.6
+pkgrel=1
+pkgdesc='VCV Rack plugin collection'
+arch=(x86_64 aarch64)
+url='https://github.com/Ahornberg/Ahornberg-VCV-Modules'
+license=(GPL3 custom:OFL)
+groups=(proaudio vcvrack-plugins)
+depends=(gcc-libs vcvrack)
```

Comment:
```con
$ find-libdeps  out/vcvrack-ahornberg-2.1.6-1-x86_64.pkg.tar.zst 
libRack.so=libRack.so-64
libOpenGL.so=0-64
libglfw.so=3-64
libjansson.so=4-64
libstdc++.so=6-64
libm.so=6-64
libgcc_s.so=1-64
libc.so=6-64
```

`OpenGL`, ` glfw` and `jansson` are covered by the dependency on `vcvrack, the question is whether these should be considered transitive dependencies or not.

`libRack.so` should be maybe added to the `depends` in the `package` function?

Reply:
This probably needs to be addressed for the other two packages as well.

Reply:
> `OpenGL`, `glfw` and `jansson` are covered by the dependency on `vcvrack`, the question is whether these should be considered transitive dependencies or not.

I'd consider them transitive dependencies, since the plugins themselves only include the `rack.hpp` header and Rack's plugin makefile links these libraries. In fact, my `plugins.patch` for the `vcvrack` package is doing that and I don't remember why all these libs were necessary for building plugins. I should definitely check that, otherwise if Rack eventually changes some dependencies we have to rebuild plugins or otherwise have invalid linked libraries in these plugins. However that would be an ABI-breaking change for Rack and would break a lot of plugins on their side as well...

> `libRack.so` should be maybe added to the `depends` in the `package` function?

Just tried that and it breaks installing the plugin, because the `libRack.so` provided by `vcvrack` is not versioned and makepkg makes the dependency `libRack.so=libRack.so-64` for some reason. I could probably go and make up a versioned soname myself in the `vcvrack` package, but that feels a bit too invasive :/

Reply:
Ok, I guess it's fine then.